### DISCUSSION TITLE
/mobile/ and /web/ images are not being created

### DISCUSSION THREAD

**Discussion by CrawlerBleak** [Retrospring/retrospring#1806] [Q&A]
I can upload a header image to my instance just fine, but when I look at my profile, it references /users/profile_headers/112/856/518/070/920/805/mobile/banner.png. /opt/retrospring/app/public/users/profile_headers/112/856/518/070/920/805/**original**/banner.png exists, but not /opt/retrospring/app/public/users/profile_headers/112/856/518/070/920/805/**mobile**/banner.png. What can I do to resolve this?

**Comment by rickymolnar** [comment]
Hey there! We were having this same issue and found out that disabling uBlock Origin solved the problem for us!
